test/run_test.py

```diff
@@ -1422,6 +1424,19 @@ def get_selected_tests(options) -> List[str]:
     if torch.version.cuda is not None:
         options.exclude.extend(["distributions/test_constraints"])
 
+    # these tests failing in Python 3.12 temporarily disabling
```

Comment:
Why are some tests disabled by hand and some disabled here?

Reply:
The tests disabled here are the ones that are failing at the beginning during first import. We also disable whole test suites here